ccmlib/dse_cluster.py

```diff
@@ -18,10 +18,12 @@
 class DseCluster(Cluster):
 
     def __init__(self, path, name, partitioner=None, install_dir=None, create_directory=True, version=None, dse_username=None, dse_password=None, dse_credentials_file=None, opscenter=None, verbose=False):
-        if dse_credentials_file:
-            self.load_credentials_from_file(dse_credentials_file)
-        else:
+        self.dse_username = None
+        self.dse_password = None
+        self.load_credentials_from_file(dse_credentials_file)
+        if dse_username is not None:
             self.dse_username = dse_username
+        if dse_password is not None:
```

Comment:
Is there a need to define self.dse_username and password as None? Couldn't this just be
```
self.load_credentials_from_file(dse_credentials_file)
if dse_username is not None:
    self.dse_username, self.dse_password = dse_username, dse_password 
```